core/src/main/java/greencity/controller/HabitAssignController.java

```diff
@@ -147,6 +147,31 @@ public ResponseEntity<List<HabitAssignDto>> getCurrentUserHabitAssignsByIdAndAcq
                 .getAllHabitAssignsByUserIdAndStatusNotCancelled(userVO.getId(), locale.getLanguage()));
     }
 
+    /**
+     * Method that return UserShoppingList and CustomShoppingList.
+     *
+     * @param habitId @link HabitVO} id.
+     * @param userVO  {@link UserVO} instance.
+     * @param locale  needed language code.
+     * @return User Shopping List and Custom Shopping List.
+     */
+    @ApiOperation(value = "Get user shopping and user custom shopping lists")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = HabitAssignDto.class,
+            responseContainer = "List"),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
```

Comment:
This endpoint is available for all user roles?
Or we could have `403` response too?